cloudberry/zion/src/main/scala/edu/uci/ics/cloudberry/zion/actor/ProgressiveSolver.scala

```diff
@@ -58,11 +58,12 @@ class ProgressiveSolver(val dataManager: ActorRef,
       val boundary = new TInterval(min, max)
 
       val initialDuration = config.FirstQueryTimeGap
+      val minimumDuration = config.MinTimeGap
```

Comment:
What if the user didn't provide this value in the config file?  If so, should we assign a default value?

Reply:
Normally the front-end assigns a default value for parameters. @baiqiushi: can you check for this parameter?

Reply:
Yes. We have default value assigned when the `config` object is initiated in `zion/src/main/scala/edu.uci.ics.cloudberry.zion/common/Config` file:
`val MinTimeGap = config.getString("berry.query.gap").map(parseTimePair).getOrElse(1 day)`.
So, for this case, no problem.